ui/components/NodeInputButton.tsx

```diff
@@ -1,46 +1,38 @@
-import { getNodeLabel } from "@ory/integrations/ui"
+import { getNodeLabel } from "@ory/integrations/ui";
 import { Button } from "@canonical/react-components";
-import { FormEvent } from "react"
+import React, { FC, FormEvent } from "react";
+import { NodeInputProps } from "./helpers";
 
-import { NodeInputProps } from "./helpers"
-
-export function NodeInputButton({
+export const NodeInputButton: FC<NodeInputProps> = ({
   node,
   attributes,
   setValue,
   disabled,
   dispatchSubmit,
-}: NodeInputProps) {
+}) => {
   // Some attributes have dynamic JavaScript - this is for example required for WebAuthn.
-  const onClick = (e: MouseEvent | FormEvent) => {
+  const handleClick = (e: MouseEvent | FormEvent) => {
     // This section is only used for WebAuthn. The script is loaded via a <script> node
     // and the functions are available on the global window level. Unfortunately, there
     // is currently no better way than executing eval / function here at this moment.
     //
     // Please note that we also need to prevent the default action from happening.
     if (attributes.onclick) {
-      e.stopPropagation()
-      e.preventDefault()
-      const run = (() => attributes.onclick)
-      run()
-      return
+      e.stopPropagation();
+      e.preventDefault();
+      const run = () => attributes.onclick; // todo wtf?
+      run();
```

Comment:
This is based on https://github.com/ory/kratos-selfservice-ui-react-nextjs/blob/master/pkg/ui/NodeInputButton.tsx#L23. Since we don't use WebAuthn, I believe it is fine to remove this.

Reply:
I'll remove it in a new PR.